scss/app/demo.scss

```diff
@@ -191,6 +191,14 @@
     background: $black87;
   }
 
+  .buttons-group--block {
+    padding: 3px;
+
+    button {
```

Comment:
Not use element selector. The button could be a link instead of a button. Use the `.button` class.